package.json

```diff
@@ -8,23 +8,30 @@
     "json",
     "bemjson"
   ],
-  "main": "lib/bemReact.js",
+  "main": "lib/index.js",
   "author": "Dmitry Filatov <dfilatov@yandex-team.ru>",
   "peerDependencies": {
     "react": "0.14.x"
   },
   "devDependencies": {
+    "babel-cli": "^6.4.5",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
@dfilatov  npm@3 does not install peerDependencies
Instead of install `npm WARN babel-preset-es2015-loose@7.0.0 requires a peer of babel-preset-es2015@*` warn will shown

npm@2  had to remind
```sh
npm WARN peerDependencies The peer dependency babel-preset-es2015@* included from babel-preset-es2015-loose will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```